### PR TITLE
Strategy merge - MANTRA DAO v1

### PR DESCRIPTION
**Pull request by spaceforce-dev**
Fixes # .

Changes proposed in this pull request:
- MANTRA DAO voting strategy
- OM staking balance merged with OM-UNIV2 
- OM-UNIV2 token has a higher weight

### CODE REVIEW

File: src/strategies/mantra-dao/index.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'Spaceforce';
```
Comment (bonustrack) [inline src/strategies/mantra-dao/index.ts:4]: This should be your github username
```suggestion
export const author = 'spaceforce-dev';
```

File: src/strategies/mantra-dao/index.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const increaceWeighting = (number) => {
+    return number * options.omUniScale
+  }
+
+  const omStakingResponse = await multicall(
```
Comment (bonustrack) [inline src/strategies/mantra-dao/index.ts:44]: Can you merge the 2 multicall in one? That will avoid sending 2 requests from user browsers and make result faster to resolve.

**Review by bonustrack — COMMENTED** [review]
Can you add a file examples.json in your strategy to make it easy to test on test/index.ts file, see example here: https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/erc20-balance-of/examples.json The example should give positive score.

File: src/strategies/mantra-dao/index.ts (hunk 2 above)
Comment (spaceforce-dev) [reply src/strategies/mantra-dao/index.ts:44]: Ok, Can you give any ideas how to do that, i'm not familiar with multi-call. Thanks for the help!

File: src/strategies/mantra-dao/index.ts (hunk 1 above)
Comment (spaceforce-dev) [reply src/strategies/mantra-dao/index.ts:4]: Ok will do

File: src/strategies/mantra-dao/index.ts (hunk 2 above)
Comment (bonustrack) [reply src/strategies/mantra-dao/index.ts:44]: You will need to merge the 2 arrays of calls together:
- This one: https://github.com/snapshot-labs/snapshot.js/pull/27/files#diff-b5ff3ff9952840d980dad2da4ac1c03ae1bee0195913c96fe1f895937990f30fR48
- And this one https://github.com/snapshot-labs/snapshot.js/pull/27/files#diff-b5ff3ff9952840d980dad2da4ac1c03ae1bee0195913c96fe1f895937990f30fR55

You can use `array1.concat(array2)` for this. Then you would do a single multicall and separate the results you get in 2 using the array length. See example here: https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/ctoken/index.ts#L92-L93

**Comment by spaceforce-dev** [comment]
Updated to come from MANTRA DAO git account 

https://github.com/snapshot-labs/snapshot.js/pull/29

**Pull request closed without merging**